public/app/features/templating/templateSrv.js

```diff
@@ -107,10 +111,15 @@ function (angular, _) {
       });
     };
 
-    this.getAllValue = function(variable) {
+    this.getAllValue = function(variable, format) {
       if (variable.allValue) {
         return variable.allValue;
       }
+
```

Comment:
Yes the point is to match All values in the variable list of options, which may not be the same as `.*`

Custom all value should work. But looking at the code, it could be a problem with the custom all value is regex, then it will be regex escaped. so might need an option to disable escaping of custom all value.